### ISSUE TITLE
Accessibility Testing for [Benefits Microservices, Chatbot, Claims Status Flow]

### ISSUE THREAD

**Issue by leahspix** [department-of-veterans-affairs/va.gov-team#79919]
### Product information

- [X] Team name, product name, and feature name have been added to the title of this issue.
- [X] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

bushnicoleVA

### Use of color and color contrast checks

- [X] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [X] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [X] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [X] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [X] Any charts, maps, infographics, and tables convey all information without only relying on color
- [X] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

The team does not have infographics in the Claims Status flow as part of the Chatbot. One of our speak tags starts with "this is a yellow alert box" but the team will remove the language that notes the alert box is yellow.

**Color contrast/ratios:** 
- The Ratio for the yellow background against the blue link is 6.02:1
- The yellow background against the standard link text is black and 18.81:1 
- All other chat boxes conform to the standard colors across the Chatbot
- The post-click link color has a contrast ratio of 9.11:1

### Who completed the axe scans?

jbackfieldVA

### axe checks

- [X] Each page has been scanned using axe (results shared below)
- [X] axe is integrated into your end-to-end testing

### axe DevTools scan results

The scan came back and stated "you have zero automated issues"

### How did axe testing go?

No issues "You have (0) automatic issues, nice!"

### Who completed the content zoom and reflow test?

bushnicoleVA

### Content zoom and reflow checks

- [X] All page elements are readable and usable at 200% zoom
- [X] All page elements are readable and usable at 300% zoom
- [X] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?

Tested on desktop and "mobile" using the developer tools and no issues identified. 

### Who completed the keyboard navigation test?

bushnicoleVA

### Keyboard navigation checks

- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [X] All elements under focus have a visible focus indicator
- [X] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

Keyboard testing was executed using JAWS. 
The Chatbot platform team ( @patilvrushali)  is in the process of working to address Skill agnostic accessibility issues that have been identified by the VA Platform Governance team and CAIA. @anitadewitt-va
